server/index.js

```diff
@@ -0,0 +1,16 @@
+const dbHelper = require('../database/index.js');
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+
+app.use(bodyParser.json());
+app.use(express.static('../client/public'));
+
+app.get('/reviews/:id', (req, res) => (
```

Comment:
Better to destructure params from your req object so you don't have to ever type req.params.id. 
```
app.get('/reviews/:id', ({ params: { id } }, res) => (
    dbHelper.getListings( id, function(results) {
```

Reply:
Updated